src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -98,7 +100,23 @@ public void ClientAsyncAuthenticate_MismatchProtocols_Fails(SslProtocols server,
             Assert.Throws(expected, () => ClientAsyncSslHelper(server, client));
         }
 
+        [Theory]
+        [MemberData("ProtocolMismatchDataSsl2Specific")]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Right, that's my confusion: it's supposed to fail, but we're disabling the test that verifies it fails.  If it's just that it fails with a different exception type, then we should have a test that verifies the expected exception type.  Though it's not clear to me why we can't make it fail with the same exception type: that should be our preferred approach if it's feasible.